exercises/en/test_03_05.py

```diff
@@ -0,0 +1,21 @@
+def test():
+    # Here we can either check objects created in the solution code, or the
+    # string value of the solution, available as __solution__. A helper for
+    # printing formatted messages is available as __msg__. See the testTemplate
+    # in the meta.json for details.
+    # If an assertion fails, the message will be displayed
+
+    # Since we haven't started assigning charts to variable names yet,
+    # this might be the better way to test for the first exercise.
+    # Maybe even for later exercises.
+    assert 'alt.Chart' in __solution__, "Make sure you are calling the 'Chart' function?"
+    assert 'mark_bar' in __solution__, "Make sure you are creating a bar plot using 'mark_bar'."
+    assert 'bin=alt.Bin' in __solution__, "Make sure you are setting the 'maxbins' parameter."
+    assert 'opacity=' in __solution__, "Make sure you are setting an appropriate opacity value"
+    assert 'culmen_depth_mm' in __solution__, "Make sure you are setting the x-axis to 'culmen_depth_mm'."
+    assert 'count()' in __solution__, "Make sure you are plotting the count on the y-axis."
+    assert "color=" in __solution__, "Make sure you are coloring by species."
```

Comment:
Check these channels so that they are like you did them in 3_04. So X, Y and Colour, Same with opacity != 1. 

To check count, Junghoo did something like this 
```
assert (penguin_bar.encoding.x.shorthand in {'count()', 'count():quantitative', 'count:Q'} or 
            penguin_bar.encoding.x.field in {'count()', 'count():quantitative', 'count:Q'}), "Make sure you are using 'count()' as the x-axis encoding."
```

Reply:
I can't do this when the plots are faceted.